recipes-bsp/u-boot/u-boot/0001-dm-core-Move-ofdata_to_platdata-call-earlier.patch

```diff
@@ -0,0 +1,49 @@
+From 336d86ebd146905cf4384912f4f27699b6e37c72 Mon Sep 17 00:00:00 2001
+From: Simon Glass <sjg@chromium.org>
+Date: Sun, 29 Dec 2019 21:19:17 -0700
+Subject: [PATCH] dm: core: Move ofdata_to_platdata() call earlier
+
+This method is supposed to extract platform data from the device tree. It
+should be done before the device itself is probed. Move it earlier in the
+device_probe() function.
+
+Signed-off-by: Simon Glass <sjg@chromium.org>
```

Comment:
Can we also add an upstream status footer?

Reply:
Patch is taken from latest u-boot tree so it's already in tree.

Reply:
Nice. Can we have that specified (backported)?